integration/test_collection.py

```diff
@@ -57,51 +57,31 @@ def client():
     client.schema.delete_all()
 
 
-def test_create_and_delete_with_no_generic(client: weaviate.Client):
+def test_create_get_and_delete(client: weaviate.Client):
     name = "TestCreateAndDeleteNoGeneric"
     collection_config = CollectionConfig(
         name=name,
         properties=[Property(name="Name", data_type=DataType.TEXT)],
         vectorizer=Vectorizer.NONE,
     )
     client.collection.create(collection_config)
-
+    client.collection.get(name)
     assert client.collection.exists(name)
     client.collection.delete(name)
     assert not client.collection.exists(name)
 
 
-def test_create_and_delete_with_dict_generic(client: weaviate.Client):
-    name = "TestCreateAndDeleteDictGeneric"
-    collection_config = CollectionConfig(
-        name=name,
-        properties=[Property(name="Name", data_type=DataType.TEXT)],
-        vectorizer=Vectorizer.NONE,
-    )
-    client.collection.create(collection_config, Dict[str, str])
-
-    client.collection.get(name, Dict[str, str])
-    assert client.collection.exists(name)
-    client.collection.delete(name)
-    assert not client.collection.exists(name)
+def test_data_with_data_model_with_dict_generic(client: weaviate.Client):
+    name = "TestDataWithDictGeneric"
+    client.collection.get(name).data.with_data_model(Dict[str, str])
```

Comment:
what does this test tell me? :) That it doesn't crash/triggers an assertion? If yes, please add a small comment `# no crash/assertion`

Reply:
My idea with some of these tests was to have them fully typed for when we get mypy functioning fully. Then that will acts as an additional layer of testing through compile-time checks